kong/dao/schemas/apis.lua

```diff
@@ -163,7 +163,8 @@ return {
     request_path = {type = "string", unique = true, func = check_request_path},
     strip_request_path = {type = "boolean", default = false},
     upstream_url = {type = "url", required = true, func = validate_upstream_url_protocol},
-    preserve_host = {type = "boolean", default = false}
+    preserve_host = {type = "boolean", default = false},
+    retries = {type = "number", default = 5, func = check_retries},
```

Comment:
Just checked it and it works as it currently is. Adding an api on 0.9.2, then restarting Kong with this branch (running the migrations), results in this api output;
```
Mashapes-MacBook-Pro-2:kong thijs$ http get localhost:8001/apis
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Type: application/json; charset=utf-8
Date: Wed, 12 Oct 2016 18:10:31 GMT
Server: kong/0.9.2
Transfer-Encoding: chunked

{
    "data": [
        {
            "created_at": 1476295721000,
            "id": "13af5ca7-a617-4e22-92b7-540fe9607e65",
            "name": "tieske",
            "preserve_host": false,
            "request_path": "/",
            "retries": 5,
            "strip_request_path": false,
            "upstream_url": "http://www.thijsschreijer.nl"
        }
    ],
    "total": 1
}
```

Retries default is now set.